.circleci/cluster/manager1_config.yaml

```diff
@@ -10,7 +10,9 @@ rabbitmq:
   ca_path: /etc/cloudify/ca.pem
   cluster_members:
     rabbit1:
-      default: QUEUE_IP
+      node_id: 'rabbit1_node_id'
```

Comment:
Is there a reason not to have the node id as the key, rather than having a separate node_id key:value pair under the dict??

Reply:
Yes, I want the hostname as the key.